app/views/curation_concerns/base/_permission_form.html.erb

```diff
@@ -3,7 +3,7 @@
 <% public_perm = true if params[:controller] == 'batch' %>
 <% registered_perm = f.object.permissions.map { |perm| perm.access if perm.agent_name == "registered"}.compact.first %>
 
-<h2><% if params[:controller] == 'batch' %>Bulk <% end %>Permissions <% if params[:controller] == 'batch' %>
+<h2><% if params[:controller] == 'batch' %>Bulk <% end %><a name="permissions_display">Permissions </a><% if params[:controller] == 'batch' %>
```

Comment:
Why don't we just put `id="permissions_display"` on this tag: https://github.com/projecthydra/sufia/blob/5f7098d70849358828680cd7468570d577098176/app/views/curation_concerns/base/_permission_form.html.erb#L18